_config.yml

```diff
@@ -65,3 +65,8 @@ teammembers:
     role: Computer Science
     role2: (undergraduate student)
     image: sinclair_hudson.jpg
+  - name: Andy Li 
```

Comment:
We are ordering by faculty/advisor/grad/undergrad, then by first name alphabetical! Feel free to move your name up :)